### PR TITLE
[C++] Add new paozhu

### PR DESCRIPTION
**Pull request by hggq**
add lost libbrotli-dev package

### CODE REVIEW

**Review by hggq — COMMENTED** [review]
This new framework can be run on my fork workflow

File: frameworks/C++/paozhu/paozhu.dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+RUN apt-get -y install libmysqlclient-dev
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+WORKDIR /
+
+RUN wget https://github.com/hggq/paozhu/releases/download/v1.5.8/benchmark.zip
```
Comment (volyrique) [inline frameworks/C++/paozhu/paozhu.dockerfile:17]: You must use the test implementations that you are adding with this pull request, not the source files that are located inside another repository.

Comment (hggq) [reply frameworks/C++/paozhu/paozhu.dockerfile]: OK, it has been modified

**Comment by hggq** [comment]
Hi @NateBrady23 Please merge this PR, thank you very much.

**Comment by NateBrady23** [comment]
@hggq Please remove the sql files and any other files that aren't necessary for the tests to run in our environment

**Comment by hggq** [comment]
> @hggq Please remove the sql files and any other files that aren't necessary for the tests to run in our environment

has remove.

File: frameworks/C++/paozhu/paozhu_benchmark/conf/server.conf [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+threadmax=1024
+threadmin=5
+httpport=8888
+httpsport=4430
+cothreadnum=8 ;Coroutines run on thread num
+
+http2_enable=0
+debug_enable=1
+deamon_enable=0
+mainhost=www.869869.com
```
[the remark this answers is not in the document]
Comment (hggq) [reply frameworks/C++/paozhu/paozhu_benchmark/conf/server.conf]: This is a locally provided test domain name certificate for the framework, which has been removed

**Pull request merged**